test/e2e/integration/004-gutenberg-examples.js

```diff
@@ -0,0 +1,25 @@
+const conditionalDescribe = Cypress.env( 'EXAMPLES' ) ? describe : describe.skip;
+conditionalDescribe( 'Example Blocks', () => {
+	before( () => {
+		cy.newPost();
+	} );
+
+	it( 'Should insert example blocks', () => {
+		cy.get( '.editor-header [aria-label="Insert block"]' ).click();
+		cy.get( '[placeholder="Search for a block"]' ).type( 'example' );
+		cy.get( '.editor-inserter__block' ).contains( 'Example' ).click();
```

Comment:
Maybe we can loop over the title of all the examples blocks and repeat the same steps for each block?

Reply:
I can absolutely support that idea.